### ISSUE TITLE
Function return value is discarded at call site

### ISSUE THREAD

**Issue by retailcoder** [rubberduck-vba/Rubberduck#2586]
We currently have an inspection that fires a result when a function's return value is *never* used anywhere, in any call site; the `Target` of that inspection is the function declaration itself.

We need another one, similar, to identify the call sites. That way if a function's return value *is* used in one place, but discarded in 20 others, we'll have inspection results for it.

**Comment by Vogel612** [comment]
Question: Are we going to offer a quick fix for this, and if yes: how?

**Comment by retailcoder** [comment]
@Vogel612 I don't think we *can*; at best we'd declare and assign a new local variable... that nothing would refer to.
